lms/templates/wiki/base.html

```diff
@@ -6,6 +6,8 @@
 {% block bodyclass %}view-incourse view-wiki{% endblock %}
 
 {% block headextra %}
+  <script type="text/javascript" src="/i18n.js"></script>
+  <script src="{% static 'js/vendor/backbone-min.js' %}"></script>
```

Comment:
See if backbone is already loaded on the page (I'm not clear if this file ends up loading base_vendor_js).